stylesheets/portfolio.css

```diff
@@ -0,0 +1,124 @@
+@import url('https://fonts.googleapis.com/css?family=Muli');
+@import url('https://fonts.googleapis.com/css2?family=Jua&display=swap');
+
+.container {
+    border: 1px;
+    border-color:cyan;
+    border-style: dotted;
+    display: grid;
+    width: 100vw;
+    min-height: 100vh;
+    grid-template-rows: 15vw 5vw auto 10vw;
+    grid-template-columns: 10vw 80vw 10vw;
+}
+
+body {
+    font-family: Muli, sans-serif;
+    color:aliceblue;
+    background-color: #8e9aaf;
+}
```

Comment:
Some of this CSS is repeated across your files. Consider having a global CSS file to place shared styling in, and only have the styling specific to each page in these other CSS files. You can import multiple CSS files in an HTML page!